VideoLocker/src/main/java/org/edx/mobile/view/VideoListActivity.java

```diff
@@ -195,10 +195,7 @@ public void run() {
             }
 
             String filepath = null;
-            // check if file available on local
-            if (video.isVideoForWebOnly) {
```

Comment:
@mdinino I believe you were supposed to revert this change. The logic is changed here.